package.json

```diff
@@ -252,6 +252,21 @@
             {
                 "view": "azureActivityLog",
                 "contents": "View all of your recent activities and quickly access resources you've recently created. \n [Create an Azure resource](command:azureResourceGroups.createResource)"
+            },
+            {
+                "view": "azureWorkspace",
+                "contents": "No workspace has been opened.",
+                "when": "azureWorkspace.state == 'noWorkspace'"
```

Comment:
Can you use the builtin `workspaceFolderCount` to replace this? e.g. `"when": "workspaceFolderCount < 1"`? Or is the goal just consistency with the other two?

Reply:
I didn't realize `workspaceFolderCount` existed, actually. :-) Still, I think I would prefer the consistency of using a specific state value.